pkg/processor/runtime/python/py/_nuclio_wrapper.py

```diff
@@ -186,7 +186,7 @@ def serve_requests(self, num_requests=None):
                     # try to json encode the response
                     encoded_response = self._json_encoder.encode(response)
 
-                except Exception as err:
+                except BaseException as err:
```

Comment:
please update for uniformity:
`except BaseException as err` -> `except BaseException as exc`
(also on line 194)